src/Common/Configuration.ts

```diff
@@ -0,0 +1,44 @@
+﻿import { config } from "mssql";
+import * as nconf from "nconf";
+
+/*
+ * App Configuration
+ */
+export class Configuration {
+    private static readonly _configKeys = {
```

Comment:
Actually, what do you think about using [connection strings](https://www.npmjs.com/package/mssql#formats) instead of this configuration object?

Here are some advantages:
1) You define your connection string on your Azure Web App (no need to put it in any files) and access it from node.js [as an environment variable](https://docs.microsoft.com/en-us/azure/app-service/web-sites-configure#connection-strings), similar to how you're accessing the port below.
2) If you decide to move to SQL hosted on Azure, you can completely eliminate passwords by using Managed Identity.

Reply:
Done.